library/std/src/io/mod.rs

```diff
@@ -2213,7 +2213,11 @@ impl<T: Read, U: Read> Read for Chain<T, U> {
 
     unsafe fn initializer(&self) -> Initializer {
         let initializer = self.first.initializer();
-        if initializer.should_initialize() { initializer } else { self.second.initializer() }
+        if initializer.should_initialize() {
```

Comment:
This change was from my RustFmt run. If you would rather keep the previous formatting, please let me know.

Reply:
Nvm, I need to keep the formatting as it was